chainquery/Dockerfile

```diff
@@ -1,52 +1,24 @@
-## This base image is for running latest chainquery
+## Get the latest source and extract it for the app container.
+## Design choices, two RUN layers intended to keep builds faster, the zipped
 FROM ubuntu:18.04 as prep
 LABEL MAINTAINER="leopere [at] nixc [dot] us"
-
-## Install everything needed to unzip the file containing Chainquery
 RUN apt-get update && \
```

Comment:
The linter is complaining about doing version pinning but it's really not necessary in this case the latest will generally be the best option.